aws-sdk-v2/src/main/java/io/micronaut/aws/sdk/v2/service/AwsClientFactory.java

```diff
@@ -60,10 +68,13 @@ protected AwsClientFactory(AwsCredentialsProviderChain credentialsProvider, AwsR
      * @return The sync builder
      */
     public SB syncBuilder(SdkHttpClient httpClient) {
-        return createSyncBuilder()
+        SB sb = createSyncBuilder()
                 .httpClient(httpClient)
                 .region(regionProvider.getRegion())
                 .credentialsProvider(credentialsProvider);
+        // Do not replace with method reference - https://bugs.openjdk.org/browse/JDK-8141508
```

Comment:
```suggestion
        // Do not replace with method reference - https://bugs.openjdk.org/browse/JDK-8141508
        //noinspection Convert2MethodRef        
```

So that, at least IDEA, stops highlighting the line.

Please add it to the other places too